torch/_torch_docs.py

```diff
@@ -7637,13 +7653,17 @@ def merge_dicts(*dicts):
 
     >>> torch.remainder(torch.tensor([-3., -2, -1, 1, 2, 3]), 2)
     tensor([ 1.,  0.,  1.,  1.,  0.,  1.])
-    >>> torch.remainder(torch.tensor([1, 2, 3, 4, 5]), 1.5)
-    tensor([ 1.0000,  0.5000,  0.0000,  1.0000,  0.5000])
+    >>> torch.remainder(torch.tensor([1, 2, 3, 4, 5]), -1.5)
+    tensor([ -0.5000, -1.0000,  0.0000, -0.5000, -1.0000 ])
 
 .. seealso::
 
         :func:`torch.fmod`, which computes the element-wise remainder of
```

Comment:
The see also section should just refer to the other function

Reply:
Do you mean it should just be the following?

```
.. seealso::

            :func:`torch.fmod`
```

I think that the existing one is better as it just gives a very brief review of what `fmod` is. But just an opinion, looking forward to hear what you think about it.

Reply:
It should include a brief description of fmod that highlights the difference with remainder (if the above docs suggestion is accepted than this would probably be something like "just computes the modulus for integer inputs and applies C++'s std::fmod for floating point inputs")